google-cloud-tools-plugin/google-cloud-core/resources/META-INF/google-cloud-core.xml

```diff
@@ -36,6 +36,10 @@
     <applicationConfigurable parentId="google.settings"
                              provider="com.google.cloud.tools.intellij.analytics.UsageTrackerConfigurableProvider"/>
 
+
+    <applicationService
+            serviceInterface="com.google.cloud.tools.intellij.analytics.UsageTrackingManagementService"
```

Comment:
nit: but just to be more inline with our current conventions, how about `UsageTrackerManager` for the interface and `DefaultUsageTrackerManager` for the impl? Users of this will be calling the getInstance from UsageTrackerManager anyway.